reckoner/chart.py

```diff
@@ -409,8 +407,11 @@ def build_files_list(self):
 
     @staticmethod
     def _interpolate_env_vars_from_string(original_string: str) -> str:
+
+        # We should never interpolate an env var in a comment. Strip comments from the string before interpolating.
+        comments_removed = re.sub(r'([^#]*)(#.*)', r"\g<1>", original_string)
```

Comment:
nit. quote style? Intentional or meaningful?

Reply:
I think I was trying to get it to work and accidentally ended up with different styles. Will fix